packages/protocol/src/action-protocol/element-selection.ts

```diff
@@ -45,14 +51,32 @@ export namespace SelectAction {
         return Action.hasKind(object, KIND) && hasArrayProp(object, 'selectedElementsIDs') && hasArrayProp(object, 'deselectedElementsIDs');
     }
 
-    export function create(options: { selectedElementsIDs?: string[]; deselectedElementsIDs?: string[] } = {}): SelectAction {
+    export function create(options: { selectedElementsIDs?: string[]; deselectedElementsIDs?: string[] | boolean } = {}): SelectAction {
+        const { deselectedElementsIDs, ...remainingOptions } = options;
         return {
             kind: KIND,
-            selectedElementsIDs: [],
-            deselectedElementsIDs: [],
-            ...options
+            selectedElementsIDs: options.selectedElementsIDs ?? [],
+            deselectedElementsIDs: isStringArray(deselectedElementsIDs, true) ? deselectedElementsIDs : [],
+            deselectAll: typeof deselectedElementsIDs === 'boolean' ? deselectedElementsIDs : false,
+            ...remainingOptions
         };
     }
+
+    export function addSelection(options: { selectedElementsIDs: string[] }): SelectAction {
+        return create(options);
+    }
+
+    export function removeSelection(options: { deselectedElementsIDs: string[] }): SelectAction {
+        return create(options);
+    }
+
+    export function setSelection(options: { selectedElementsIDs: string[] }): SelectAction {
+        return create({ ...options, deselectedElementsIDs: true });
+    }
+
+    export function deselectAll(options: object = {}): SelectAction {
```

Comment:
Is this really needed. Isn't this allready handled by  `SelectAllAction.create(false)` ?

Reply:
Good point! I didn't think of the `SelectAll`. I removed this method from here and the feedback selection.